### ISSUE TITLE
Add ability to ignore files during linting

### ISSUE THREAD

**Issue by pdehaan** [mozilla/addons-linter#2942]
### Describe the problem and steps to reproduce it:

Ref: https://github.com/mozilla/web-ext/issues/1770

When using `web-ext`, it looks like I can ignore files using an `--ignore-files=""` flag. I can't find a similar flag/API for the `addons-linter`.

### What happened?

No API.

### What did you expect to happen?

API for file ignoring (since I want to ignore a 3rd party vendor file).

### Anything else we should know?

```sh
npx addons-linter --version # 1.14.0
```

**Comment by willdurand** [comment]
@rpl WDYT?

**Comment by rpl** [comment]
@willdurand @pdehaan web-ext does integrate addons-linter as a library and to implement its `--ignore-files` cli option is passing to the addons-linter a function through the `shouldScanFile` option, and so `shouldScanfile` is technically the API to achieve the same thing when using the addons-linter npm package as an npm module in other projects.

It looks that there is also an explicit mention to the `shouldScanfile` and to `scanFile` options in the README.md:
- https://github.com/mozilla/addons-linter#linter-api-usage

And so this should already be supported, maybe we should tweak that section in the readme to make it more visible?

**Comment by willdurand** [comment]
> And so this should already be supported, maybe we should tweak that section in the readme to make it more visible?

Right, updating the README sounds good.

**Comment by willdurand** [comment]
> > And so this should already be supported, maybe we should tweak that section in the readme to make it more visible?
> 
> Right, updating the README sounds good.

S because of this ^

**Comment by OluchiEzeifedikwa** [comment]
Hello, I would like to contribute as my first contribution to this if its still available.
